### PR TITLE
Replicate files not only file metadata

### PR DESCRIPTION
**Pull request by erlendvollset**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
#### Checkmarx scan completed with the following findings
|Lines|Severity|Category|File|Link|
---|---|---|---|---

**Comment by gaetan-h** [comment]
Looks good. Please note that we are still having issues with the CI/CD part. 
We are trying to move to Github actions. 

I am almost finished testing, if you want to assist, please review
https://github.com/cognitedata/cognite-replicator/pull/140

Maybe best to merge yours when that is successful :)

**Review by gaetan-h — APPROVED** [review]
Looks good. Can be merged :)

File: cognite/replicator/files.py [hunk 1]
```diff
@@ -65,13 +66,29 @@ def update_file(
     return dst_file
 
 
+def create_and_upload_file(file_metadata: FileMetadata, client_src: CogniteClient, client_dst: CogniteClient) -> None:
+    file_bytes = client_src.files.download_bytes(external_id=file_metadata.external_id)
+    client_dst.upload(
```
Comment (kornelione) [inline cognite/replicator/files.py:71]: client_dst.files.upload

**Review by kornelione — APPROVED** [review]

**Pull request closed without merging**